tokio-trace-futures/Cargo.toml

```diff
@@ -4,13 +4,13 @@ version = "0.0.1"
 authors = ["Eliza Weisman <eliza@buoyant.io>"]
 
 [features]
-default = ["with-tokio"]
-with-tokio = ["tokio"]
+default = ["tokio"]
```

Comment:
do you need to add executor here too? or have it in another feature?

Reply:
it's not on by default. all optional dependencies are their own feature flags (which I had forgotten when i made the initial "with-tokio" feature).